hack/pin-dependency.sh

```diff
@@ -29,8 +29,6 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 # Explicitly opt into go modules, even though we're inside a GOPATH directory
 export GO111MODULE=on
-# Explicitly clear GOPATH, to ensure nothing this script calls makes use of that path info
```

Comment:
unintentional slip?

Reply:
intentional, caused the same problems that were fixed in https://github.com/kubernetes/kubernetes/pull/76511